sdcflows/utils/tools.py

```diff
@@ -154,3 +154,59 @@ def brain_masker(in_file, out_file=None, padding=5):
     img.__class__(data, img.affine, img.header).to_filename(out_brain)
 
     return str(out_brain), str(out_probseg), str(out_mask)
+
+
+def reorient_pedir(pe_dir, source_ornt=None, target_ornt=None):
```

Comment:
Not dying on this hill, but I find whenever a function has 2 optional parameters that can either be defined, and could effect the output, I tend to make them keyword only to remove the ambiguity.

Reply:
I think including that first bit in the docstring would go a long way for clarity. I tend to prefer the verbosity of `reorient_pedir(pe_dir, source_ornt=None, target_ornt=ornt)`, but I'll leave it for you to judge.